app/pages/lab/project.cjsx

```diff
@@ -252,7 +252,7 @@ module.exports = React.createClass
           <p className="form-help">Loading project</p>
         </div>
       } then={([project, owners]) =>
-        if @props.user in owners
+        if @props.user in owners or @props.user.admin
```

Comment:
`user.admin` just indicates that a user can _access_ admin mode.

`apiClient.params.admin` indicates if admin mode is actually switched on.